packages/next/client/image.tsx

```diff
@@ -638,6 +656,22 @@ export default function Image({
     })
   }
 
+  let isRaw = false
+  const rawStyle: ImgElementStyle = {}
+
+  if (layout === 'raw') {
+    isRaw = true
+    const styleProp = 'style' in rest && rest.style ? rest.style : {}
+    Object.assign(
+      rawStyle,
+      {
+        ...(objectFit ? { objectFit } : {}),
```

Comment:
I chose to pass through objectFit and objectPosition as styles to the img. This isn't specifically called out in the documentation, but it feels like it would be a common expectation that if `objectFit="cover"` works on images in the `fill` layout, it should work in the `raw` layout. I'm open to changing this though